sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -100,6 +97,54 @@ kernel_pid_t gnrc_ipv6_init(void)
     return gnrc_ipv6_pid;
 }
 
+static bool _dispatch_next_header(gnrc_pktsnip_t *current,
```

Comment:
Sorry, returning `bool` was redundant. We might save a few bytes by changing it to `void`.